lib/resolver.content.js

```diff
@@ -148,16 +147,27 @@ module.exports = class PodletClientContentResolver {
                         `remote resource responded with non 200 http status code for content - code: ${response.statusCode} - resource: ${outgoing.name} - url: ${outgoing.contentUri}`,
                     );
 
-                    reject(
-                        boom.boomify(
-                            new Error(
-                                `Could not read content. Resource responded with ${response.statusCode} on ${outgoing.contentUri}`,
-                            ),
-                            {
-                                statusCode: response.statusCode,
-                            },
-                        ),
-                    );
+                    const isRedirect =
+                        response.statusCode >= 300 && response.statusCode < 400;
+
+                    const statusCode = isRedirect ? 500 : response.statusCode;
+
+                    const errorMessage = `Could not read content. Resource responded with ${response.statusCode} on ${outgoing.contentUri}`;
+
+                    const errorOptions = {
+                        statusCode,
+                        decorate: {
+                            isRedirect,
+                            statusCode: response.statusCode,
```

Comment:
boom decoration gets applied to the root of the error so this statusCode will be available at `err.statusCode`